tests/test_kernels.py

```diff
@@ -178,13 +200,13 @@ def test_periodic_1d_and_2d(D, N, lengthscale, variance, period):
 
 
 kernel_setups = [
-                    kernel() for kernel in gpflow.kernels.Stationary.__subclasses__()
-                ] + [
-                    gpflow.kernels.Constant(),
-                    gpflow.kernels.Linear(),
-                    gpflow.kernels.Polynomial(),
-                    gpflow.kernels.ArcCosine()
-                ]
+    kernel() for kernel in gpflow.kernels.Stationary.__subclasses__()
```

Comment:
As far as I understand that this change was caused by your IDE? Is that right?

Reply:
No, I manually removed the superfluous space to be in keeping with the other `kernel_setups_*`